advanced_start/index.rst

```diff
@@ -0,0 +1,15 @@
+.. _advanced_start_index:
+
+******************
+Advanced Tutorial
+******************
+
+In this section, we're going to productionise a Random Forest classifier written with `sklearn`, deploy it to the cloud, and use it in a more sophisticated workflow. 
```

Comment:
Good point on the former. On the latter, it's quite a common term but is a bit clunky. Do you prefer operationalise?